Emurgo/yoroi-mobile
# feat(swap): Integrate open orders with MuesliSwap API

Resolves [YOMO-628](https://emurgo.atlassian.net/jira/software/c/projects/YOMO/issues/YOMO-628)

To test the functionality, you may need to
1. Have a mainnet wallet with some ADA
2. Place limit order on muesli swap web
3. Wait for the transaction to appear on the mobile
4. Navigate to swap
5. Navigate to open orders
6. You should see your open order

[YOMO-628]: https://emurgo.atlassian.net/browse/YOMO-628?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
